jscomp/bsb/bsb_db_encode.ml

```diff
@@ -112,7 +112,7 @@ let write_build_cache buf (bs_files : Bsb_db.t) : unit =
   in
 
   let bsbuild_rule =
-    Format.asprintf "@\n(rule (with-stdout-to %s (run echo -n \"%s\")))"
+    Format.asprintf "@\n(rule (write-file %s %s))"
```

Comment:
The reason we don’t need to quote  here is: the contents are base64 serialized, so there are no special chars that need to be quoted.